Explorer/Assets/DCL/Multiplayer/Profiles/Entities/RemoteEntities.cs

```diff
@@ -50,7 +51,8 @@ public RemoteEntities(
 
         public void Initialize(RemoteAvatarCollider remoteAvatarCollider)
         {
-            remoteAvatarColliderPool = componentPoolsRegistry.AddGameObjectPool(() => Object.Instantiate(remoteAvatarCollider, Vector3.zero, Quaternion.identity));
+            RemoteEntitiesParent = new GameObject("REMOTE_ENTITIES").transform;
```

Comment:
Does this need cleaning up in Dispose?

Reply:
Ahh you are right, we are not destroying these on Dispose!